src/styles/components/_details.scss

```diff
@@ -58,7 +58,7 @@ $BORDER_COLOR: $GREY_300;
 .w-details__header {
   color: $LIGHT_BLUE_600;
   margin: 0;
-  padding: .75rem #{$CHEVRON_SIZE + 12px} .75rem 0;
+  padding: .75rem #{$CHEVRON_SIZE + 2px + 12px} .75rem 0;
```

Comment:
Initially I was thinking of changing this to `+ 14px` but I figured you were explicit about the values to hint to the reader that they relate to other parts of the code, is that correct?

Reply:
Yeah, it's padding + font-size for the chevron pseudo-element. (Padding changes based on media query to make this fit better on feature phones.) I'll comment it.